gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagStateStoreWithDagNodes.java

```diff
@@ -33,9 +33,22 @@
  * {@link DagStateStore#cleanUp(DagManager.DagId)} should be used to delete all the {@link Dag.DagNode}s for a {@link Dag}.
  */
 public interface DagStateStoreWithDagNodes extends DagStateStore {
+
+  /**
+   * updates a dag node identified by the provided {@link org.apache.gobblin.service.modules.orchestration.DagManager.DagId}
```

Comment:
nit can you capitalize the first word of these docs. Explain what the int it returns represents